ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -35,18 +52,39 @@ worker {
             .map((tag) => `"${tag?.trim()}"`)
             .join(', ')
         : '"<tag1>", "<tag2>"'
-    }]
+    }],
+  },
+  ${
+    this.features.isEnabled('byow-pki-upstream')
+      ? `initial_upstreams = [${
+          this.initialUpstreams
+            ? this.initialUpstreams
+                .split(',')
+                .filter((tag) => tag.trim())
+                .map((tag) => `"${tag?.trim()}"`)
+                .join(', ')
```

Comment:
These aren't tags anymore, I think we should just extract out a utility function to handle this since it's the exact same code.

Reply:
@ZedLi fixed